src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
Thanks for explaining me this principle in detail.

However, I don't see any value of moving 2 lines of `moment.utc(x).local().format('lll')` to the separate method in the new module. I don't know for sure if we will have third occurrence of this format in the near future.